src/Relude/Foldable/Fold.hs

```diff
@@ -1,6 +1,6 @@
 {-# LANGUAGE DataKinds            #-}
-{-# LANGUAGE ExplicitForAll       #-}
 {-# LANGUAGE PolyKinds            #-}
+{-# LANGUAGE ScopedTypeVariables  #-}
```

Comment:
`ScopedTypeVariables` are enabled by default in the `.cabal` file so this extension is redundant here

Reply:
I wonder why `ExplicitForAll` was here then, since it's implied by this.

Reply:
Probably `ScopedTypeVariables` was introduced by default in some past but weren't removed from every module where they are redundant. Because GHC doesn't warn about them so it's a manual human effort...